### PR TITLE
Updates to asset-transfer-basic Gateway sample to align with docs

### PR DESCRIPTION
**Pull request by bestbeforetoday**
Also install goimports globally to run the linting check rather than installing to each gomodule, which was causing dependency conflicts.

### CODE REVIEW

File: asset-transfer-basic/application-gateway-typescript/src/app.ts [hunk 1]
```diff
@@ -4,17 +4,38 @@
...
+//         ===> from directory /fabric-samples/test-network
+//         ./network.sh up createChannel
+// - Use any of the asset-transfer-basic chaincodes deployed on the channel "mychannel"
+//   with the chaincode name of "basic". The following deploy command will package,
+//   install, approve, and commit the javascript chaincode, all the actions it takes
+//   to deploy a chaincode to a channel.
+//         ===> from directory /fabric-samples/test-network
+//         ./network.sh deployCC -ccn basic -ccp ../asset-transfer-basic/chaincode-typescript/ -ccl typescript
+// - Be sure that Node.js is installed
+//         ===> from directory /fabric-samples/asset-transfer-basic/application-typescript
```
Comment (denyeart) [inline asset-transfer-basic/application-gateway-typescript/src/app.ts:18]:
```suggestion
//         ===> from directory /fabric-samples/asset-transfer-basic/application-gateway-typescript
```

File: asset-transfer-basic/application-gateway-typescript/src/app.ts [hunk 2]
```diff
@@ -4,17 +4,38 @@
...
+//         node -v
+// - npm installed code dependencies
+//         ===> from directory /fabric-samples/asset-transfer-basic/application-typescript
```
Comment (denyeart) [inline asset-transfer-basic/application-gateway-typescript/src/app.ts:21]: same

File: asset-transfer-basic/application-gateway-typescript/src/app.ts [hunk 3]
```diff
@@ -4,17 +4,38 @@
...
+//         npm install
+// - to run this test application
+//         ===> from directory /fabric-samples/asset-transfer-basic/application-typescript
```
Comment (denyeart) [inline asset-transfer-basic/application-gateway-typescript/src/app.ts:24]: same

File: asset-transfer-basic/application-gateway-typescript/src/app.ts (hunk 1 above)
Comment (bestbeforetoday) [reply asset-transfer-basic/application-gateway-typescript/src/app.ts:18]: I didn't change this text, just moved its location in the file. Changes and the build fix will need to wait for another day

**Review by mbwhite — APPROVED** [review]

**Pull request merged**